ROADMAP.md

```diff
@@ -0,0 +1,46 @@
+# Declarative Gradle Roadmap
+
+Declarative Gradle is an experimental project under active development.
+Below, you can see the key milestones we anticipate.
+All the dates and the scope are aspirational.
+
+To  discuss the roadmap and the related initiatives, use the
+`#declarative-gradle` channel on the [Gradle Community Slack](https://gradle.org/slack-invite)
+
+## Project Board
+
+We have a [project board](https://github.com/orgs/gradle/projects/96) published on GitHub Project.
+It captures the main initiatives and EPICs in this project.
+
+## Key Milestones
+
+### First Alpha Release
+
+Scope: Declarative Gradle can be evaluated with limited functionality and with support from the team.
+This includes:
+the alpha release of the Declarative Gradle plugins,
```

Comment:
💭 bulleted list here?

Reply:
I would not do that, unless we can reference items